### PR TITLE
Add `gatsby-transformer-contentful-richtext`

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
Add `gatsby-transformer-contentful-richtext`
Preview: 👉 https://5c04e8120097d47b3409bbbe--loudmouth.netlify.com/blog/suckaz-post

### CODE REVIEW

File: packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      actions,
+      createNodeId,
+    },
+    { ...additionalParameters }
+    )
+  })
+}
+describe(`HTML is generated correctly`, () => {
+  const content = {
+    nodeType: 'document',
```
Comment (sbezludny) [inline packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js]:
```suggestion
    nodeType: 'document',
    data: {},
```

File: packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    content: [
+      {
+        nodeType: 'paragraph',
```
Comment (sbezludny) [inline packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js]:
```suggestion
        nodeType: 'paragraph',
        data: {},
```

File: packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        content: [
+          {
+            nodeType: 'text',
+            value: 'Hello world!',
+            marks: [],
```
Comment (sbezludny) [inline packages/gatsby-transformer-contentful-richtext/src/__tests__/extend-node.js]:
```suggestion
            marks: [],
            data: {}
```

**Comment by sbezludny** [comment]
added data fields where missing

File: packages/rich-text-from-markdown/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/rich-text-from-markdown",
-  "version": "12.0.0",
+  "version": "11.0.0",
```
[the remark this answers is not in the document]
Comment (loudmouth) [reply packages/rich-text-from-markdown/package.json]: @sbezludny this is rebased from master...can you share what this line should look like?

**Pull request merged**